.github/workflows/dhis2-verify-app.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     call-workflow-e2e-prod:
         if: "!contains(github.event.head_commit.message, '[skip ci]')"
         needs: [build, lint, test]
-        uses: dhis2/data-visualizer-app/.github/workflows/e2e-prod.yml@dev
+        uses: dhis2/line-listing-app/.github/workflows/e2e-prod.yml@master
         secrets:
             baseurl: ${{ secrets.CYPRESS_DHIS2_BASE_URL_DEV }}
```

Comment:
@HendrikThePendric Just a side note, I don't think `baseurl` is actually used in the shared workflow (even though it lists it as required).

Reply:
@jenniferarnesen I think the action that computes the versions uses it right?

Reply:
I don't think so. If you search for `baseurl`, it's only occurrence is in the list of required vars: https://github.com/dhis2/line-listing-app/blob/master/.github/workflows/e2e-prod.yml

Reply:
I thought it was needed by the `compute-dev-version` job in `e2e-dev.yml`:
https://github.com/dhis2/line-listing-app/blob/db3d74809be0677c05a85a5a2a47947616fdc22c/.github/workflows/e2e-dev.yml#L24-L36

And that the shared action which is being used requires it:
https://github.com/dhis2/action-instance-version/blob/main/action.yml#L4-L6

Reply:
Oh hold up. I see what you mean now.... `e2e-prod` does *not* need it. `e2e-dev` does, but we're not talking about that file. Sorry.

Reply:
I'll just keep it there for now. I dream of a time when we have a bunch of shared workflows for all our apps. Hope it will be this autumn.